app/code/Magento/Inventory/Test/Acceptance/Test/NotifyQuantityUseDefaultAppliedToSimpleProductOnConfigurationAdminPage.xml

```diff
@@ -0,0 +1,141 @@
+            <waitForPageLoad time="10" stepKey="waitForPageLoad4"/>
+            <click selector="{{AdminEditStockSourcesSection.assignSources}}" stepKey="clickOnAssignSources1"/>
+            <actionGroup ref="searchAdminDataGridByKeyword" stepKey="searchByKeyword2">
+                <argument name="keyword" value="$$createSource1.source[name]$$"/>
+            </actionGroup>
+            <click selector="{{AdminGridRow.checkboxByValue($$createSource1.source[name]$$)}}" stepKey="clickOnTheCheckboxForSource2"/>
+            <click selector="{{AdminManageSourcesGridControls.done}}" stepKey="clickOnDone1"/>
+            <click selector="{{AdminGridMainControls.saveAndContinue}}" stepKey="saveStock1"/>
+        </before>
+
+        <after>
+            <deleteData createDataKey="simpleCategory1" stepKey="deleteCategory1"/>
+
+            <comment userInput=" Disable source, as it can brake single source mode tests." stepKey="disableSourceComment1"/>
+            <amOnPage url="{{AdminManageSourcePage.url}}" stepKey="amOnTheSourcesListPage1"/>
+            <conditionalClick selector="{{AdminDataGridHeaderSection.clearFilters}}" dependentSelector="{{AdminDataGridHeaderSection.clearFilters}}" visible="true" stepKey="clickClearFilters1"/>
+            <actionGroup ref="AdminGridFilterSearchResultsByInput" stepKey="filterSearchResults1">
+                <argument name="selector" value="AdminManageSourcesGridFilterControls.code"/>
+                <argument name="value" value="_minimalSource.source_code"/>
+            </actionGroup>
+            <click selector="{{AdminGridRow.editByValue($$createSource1.source[source_code]$$)}}" stepKey="clickOnEdit1"/>
+            <waitForPageLoad time="30" stepKey="waitForPageLoad5"/>
+            <click selector="{{AdminEditSourceGeneralSection.IsEnabledLabel}}" stepKey="disableSource1"/>
+            <actionGroup ref="AdminFormSaveAndClose" stepKey="saveAndClose1"/>
+
+            <comment userInput=" Assign Default Stock to Main Website " stepKey="assignDefaultStockToMainWebsiteComment1"/>
+            <amOnPage url="{{AdminManageStockPage.url}}" stepKey="amOnTheStocksListPage2"/>
+            <waitForPageLoad time="30" stepKey="waitForPageLoad6"/>
+            <actionGroup ref="searchAdminDataGridByKeyword" stepKey="searchByKeyword3">
+                <argument name="keyword" value="_defaultStock.name"/>
+            </actionGroup>
+            <click selector="{{AdminGridRow.editByValue(_defaultStock.name)}}" stepKey="clickOnEditForStock2"/>
+            <waitForPageLoad time="30" stepKey="waitForPageLoad7"/>
+            <selectOption selector="{{AdminEditStockSalesChannelsSection.websites}}" userInput="Main Website" stepKey="selectWebsiteAsSalesChannel2"/>
+            <click selector="{{AdminGridMainControls.saveAndContinue}}" stepKey="saveStock2"/>
+
+            <comment userInput="Set 'Notify for Quantity Below' to use system value by default." stepKey="revertNotifyQtyBelowComment1"/>
```

Comment:
I'd recommend updating this value using the ./bin/Magento CLI command instead (MFTF DevDocs):

`<magentoCLI command="config:set cataloginventory/item_options/notify_stock_qty 1" stepKey="setNotifyQty1"/>`